compiler-access/src/main/scala/org/alephium/ralph/lsp/access/compiler/message/error/StringError.scala

```diff
@@ -1,12 +1,13 @@
 package org.alephium.ralph.lsp.access.compiler.message.error
 
-import org.alephium.ralph.lsp.access.compiler.message.{CompilerMessage, SourceIndex}
+import org.alephium.ralph.SourceIndex
+import org.alephium.ralph.lsp.access.compiler.message.{CompilerMessage, SourceIndexExtra}
 
 object StringError {
   @inline def apply(message: String): StringError =
     StringError(
       message = message,
-      index = SourceIndex.empty
+      index = SourceIndexExtra.zero
```

Comment:
why not just keeping `SourceIndex.empty`?

Reply:
`SourceIndex.empty` creates a new instance on every call. `SourceIndexExtra.zero` is final.

Reply:
ooooh indeed, I guess  I'll make it final in the node too.

Reply:
That'll be great. Thank you.